rifServices/src/main/java/org/sahsu/rif/services/graphics/RIFStyle.java

```diff
@@ -307,7 +307,7 @@ else if (rifMethod.equals("standardDeviation")) {
 				groups = (Classifier) classify.evaluate(featureCollection);  // Classify data 
 			}
 			catch (NullPointerException npe) {
-				throw new Exception("Failed to classify fesatureCollection; function Name: " + classifyFunctionName +
+				throw new Exception("Failed to classify featureCollection; function Name: " + classifyFunctionName +
 					"; column: " + columnName + 
 					"; NullPointerException in classify.evaluate()");
```

Comment:
Include the caught Exception in the constructor of the new Exception. It might have useful info.